testapi.pm

```diff
@@ -213,6 +218,26 @@ sub record_info {
     $autotest::current_test->record_resultfile($title, $output, %nargs);
 }
 
+=head2 force_soft_failure
+
+=for stopwords softfail
+
+  force_soft_failure([$reason]);
+
+Like C<record_soft_failure> but overrides the test module status to a softfail
```

Comment:
your wording still sounds like you can pass this before the fail - but it's not true as it only overwrites past failures.

Reply:
I'll merge so we can deploy it today, but I would still like you to update the docu about it (or remove it if your experiments fails :)

Reply:
https://github.com/os-autoinst/os-autoinst/pull/1062